src/registry.rs

```diff
@@ -734,12 +734,20 @@ impl Diagnostic {
 }
 
 /// Pairs of checks that shouldn't be enabled together.
-pub const INCOMPATIBLE_CODES: &[(Rule, Rule, &str)] = &[(
-    Rule::OneBlankLineBeforeClass,
-    Rule::NoBlankLineBeforeClass,
-    "`D203` (OneBlankLineBeforeClass) and `D211` (NoBlankLinesBeforeClass) are incompatible. \
-     Consider adding `D203` to `ignore`.",
-)];
+pub const INCOMPATIBLE_CODES: &[(Rule, Rule, &str)] = &[
+    (
+        Rule::OneBlankLineBeforeClass,
+        Rule::NoBlankLineBeforeClass,
+        "`one-blank-line-before-class` (D203) and `no-blank-line-before-class` (D211) are \
```

Comment:
I'm so tempted to just get rid of `D203` :joy: It causes so many problems, and it's not included in any of the default conventions!